components/brave_wallet_ui/common/async/handlers.ts

```diff
@@ -87,8 +87,6 @@ async function refreshWalletInfo (store: Store) {
   const mmResult = await braveWalletService.isMetaMaskInstalled()
   store.dispatch(WalletActions.setMetaMaskInstalled(mmResult.installed))
 
-  await store.dispatch(refreshBalancesAndPrices(currentNetwork))
-  await store.dispatch(refreshTokenPriceHistory(state.selectedPortfolioTimeline))
```

Comment:
Can you double-check if there are no callers of `refreshWalletInfo()` that actually rely on `refreshBalancesAndPrices()` / `refreshTokenPriceHistory()` being called?

Reply:
Did a double check and looks good also 
`refreshWalletInfo` calls `refreshKeyringInfo` which calls `initialized`